modules/ncspot/hm.nix

```diff
@@ -1,34 +1,54 @@
-{ config, lib, ... }:
+{ mkTarget, lib, ... }:
+mkTarget {
+  name = "ncspot";
+  humanName = "Ncspot";
+  extraOptions = {
+    background = lib.mkOption {
+      type = lib.types.nullOr lib.types.str;
+      internal = true;
+      default = null;
+    };
```

Comment:
Even though it's internal, it'd still be useful to have a `description`. This could be used to explain why the option exists at all.